src/fabric/test/fabric2_tx_options_tests.erl

```diff
@@ -101,9 +103,40 @@ can_configure_options_at_runtime(_) ->
     ok = fabric2_db:delete(DbName, [?ADMIN_CTX]).
 
 
+can_apply_options_to_db_name_transactions(_) ->
+    DbName = ?tempdb(),
+
+    TxFun = fun(TxDb) ->
+        #{tx := Tx} = TxDb,
+        fabric2_fdb:create(TxDb, [?ADMIN_CTX]),
+        erlfdb:wait(erlfdb:get(Tx, <<16#FF, "/primaryDatacenter">>))
+    end,
+    TxOpts = #{read_system_keys => <<>>},
```

Comment:
One thing for future improvements will be to figure out a better UI for these options that are booleans but expect an empty binary. I don't think that work belongs as part of this PR but it'd be nice to figure something out in the future.

Reply:
yeah that was a bit confusing, I had set it to `true` initially but we're following the C API pretty closely